tests/basic_correctness/test_preemption.py

```diff
@@ -24,6 +25,13 @@
     "tests/basic_correctness/test_preemption.py`")
 
 
+@pytest.fixture
+def worker_use_ray() -> bool:
+    # When SPMD worker is used, use ray_use_worker=True
+    # to test delta input optimization works with preemption.
+    return envs.VLLM_USE_RAY_SPMD_WORKER
```

Comment:
I'm confused. How is this related with `worker_use_ray` and `distributed_executor_backend=ray` ?

Reply:
ah, I wanted to test preemption + spmd. And right now, only tp > 1 works with spmd (because tp==1, we don't have any worker by default). if worker_use_ray=True, it starts a ray worker with tp == 1, so we can test it without using tp 2.

Reply:
but actually I forgot to update test-pipeline.yaml! I am also open for suggestino for cleaner tests..